src/Umbraco.Cms.Persistence.Sqlite/Services/SqliteDistributedLockingMechanism.cs

```diff
@@ -142,7 +143,7 @@ private void ObtainWriteLock()
                     "SqliteDistributedLockingMechanism requires a transaction to function.");
             }
 
-            var query = @$"UPDATE umbracoLock SET value = (CASE WHEN (value=1) THEN -1 ELSE 1 END) WHERE id = {LockId}";
+            var query = @$"UPDATE umbracoLock SET value = (CASE WHEN (value=1) THEN -1 ELSE 1 END) WHERE id = {LockId.ToString(CultureInfo.InvariantCulture)}";
```

Comment:
FYI: you can use `FormattableString.Invariant($"")` to ensure all items are formatted using the invariant culture. In this case it's only a single item and won't make the code much shorter, but it might help readability slightly 😄